content/F2015/F15-event-dropbox-coffee-chats.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug: dropbox-coffee-chats-f15
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
```

Comment:
That's reads the same as keeping all the "and"s. How about "Come and chat with female engineers from Dopbox over coffee an snacks!"